packages/vite/index.js

```diff
@@ -19,15 +19,29 @@ function showErrorOverlay(err) {
 window.addEventListener("error", showErrorOverlay);
 window.addEventListener("unhandledrejection", (e) => showErrorOverlay(e.reason));
 
-import.meta.hot.accept("${src}", (newModule) => {
-	import.meta.hot.data.render = newModule.render;
+async function getRender(newModule) {
+        let newRender == newModule.render;
+        if (newRender) {
+            console.warn('[anywidget] Deprecation Warning. Direct export of a \'render\' will likely be deprecated in the future.')
+            return newRender;
+        }
+        newRender = newModule.default.render;
+	assert(
```

Comment:
Have you tested the plugin locally? I would assume this would raise a `ReferenceError` since `assert` is not a global.

Reply:
No need to apologize! I really really appreciate the contribution. Just wanted to make sure.

You can also test out the "local" version without much ceremony by pointing to `index.js`

```js
// vite.config.js
import anywidget from "./path/to/anywidget/packages/vite/index.js";

export default {
  plugins: [anywidget()]
}
```